docs/resources/networking_secgroup_rule.md

```diff
@@ -50,11 +49,9 @@ The following arguments are supported:
   __icmp__ and __icmpv6__. If omitted, the protocol means that all protocols are supported.
   This is required if you want to specify a port range. Changing this creates a new security group rule.
 
-* `port_range_min` - (Optional, Int, ForceNew) Specifies the lower part of the allowed port range, valid integer value
-  needs to be between 1 and 65535. Changing this creates a new security group rule.
-
-* `port_range_max` - (Optional, Int, ForceNew) Specifies the higher part of the allowed port range, valid integer value
-  needs to be between 1 and 65535. Changing this creates a new security group rule.
+* `ports` - (Optional, String, ForceNew) Specifies the allowed port value range, which supports single port (80),
```

Comment:
I'd like to make it compatible with `port_range_min` and `port_range_max`, it's OK to just directly update here but need to be compatible in codes. `port_range_min` and `port_range_max` can be converted to singile port (80) or  continuous port (1-30).

Reply:
OFC